test_unstructured/partition/pdf_image/test_pdf_image_utils.py

```diff
@@ -161,7 +163,12 @@ def test_save_elements_with_output_dir_path_none():
         )
 
         # Verify that the images are saved in the expected directory
-        expected_output_dir = os.path.join(tmpdir, "figures")
+        if storage_enabled:
```

Comment:
personally I'd prefer to see more usages of pathlib, but I believe it's not the mail goal of this PR, just a side note